dtfs-central-api/src/helpers/fee-record-payment-entity-group.helper.test.ts

```diff
@@ -0,0 +1,338 @@
+import { difference } from 'lodash';
+import { when } from 'jest-when';
+import {
+  FEE_RECORD_STATUS,
+  FeeRecordEntityMockBuilder,
+  PaymentEntityMockBuilder,
+  UtilisationReportEntity,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import {
+  FeeRecordPaymentEntityGroup,
+  getFeeRecordPaymentEntityGroups,
+  getFeeRecordPaymentEntityGroupStatus,
+  getFeeRecordPaymentEntityGroupReconciliationData,
+} from './fee-record-payment-entity-group.helper';
+import { TfmUsersRepo } from '../repositories/tfm-users-repo';
+import { aTfmUser } from '../../test-helpers';
+
+describe('fee-record-payment-entity-group.helper', () => {
```

Comment:
Thanks @JoshBinns2000 sorry I meant in the file name as well please 🙏

Reply:
I believe I have also updated the file names - I just replaced `.helper` with `-helpers` because keeping the name as just `fee-record-payment-entity-group.ts` felt like it was missing any context